beginner_source/onnx/onnx_registry_tutorial.py

```diff
@@ -0,0 +1,469 @@
+# -*- coding: utf-8 -*-
+
+"""
+`Introduction to ONNX <intro_onnx.html>`_ ||
+`Export a PyTorch model to ONNX <export_simple_model_to_onnx_tutorial.html>`_ ||
+**Introduction to ONNX Registry**
+
+Introduction to ONNX Registry
+=============================
+
+**Authors:** Ti-Tai Wang (titaiwang@microsoft.com)
+"""
+
+
+###############################################################################
+# Overview
+# ~~~~~~~~
+#
+# This tutorial is an introduction to ONNX registry, which
+# empowers users to create their own ONNX registries enabling
+# them to address unsupported operators in ONNX.
+#
+# In this tutorial, we will cover the following scenarios:
+#
+# * Unsupported ATen operators
+# * Unsupported ATen operators with existing ONNX Runtime support
+# * Unsupported PyTorch operators with no ONNX Runtime support
+#
+
+
+import torch
+print(torch.__version__)
+torch.manual_seed(191009)  # set the seed for reproducibility
+
+import onnxscript  # pip install onnxscript
+print(onnxscript.__version__)
+
+# NOTE: opset18 is the only version of ONNX operators we are
+# using in torch.onnx.dynamo_export for now.
+from onnxscript import opset18
+
+import onnxruntime  # pip install onnxruntime
+print(onnxruntime.__version__)
```

Comment:
```suggestion
```

I still don't see the value on this check in the middle of a series of tutorials, which share a common installation step for all tutorials.

If any check should be done, we should do it in the common installation step (proposed this change above in another comment)

It is repetitive to have the same dependency being checked for each tutorial again and again

Reply:
I think version is especially important in registry tutorial because:
1. PyTorch calls graph api from onnxscript
2. torchlib depends on onnxscript version
3. onnxruntime supports varies across version

If we move this to intro_onnx.py, there will be only this code (I see no reason why we should coment "the version", instead of utilizing print out.), and I think intro_onnx.py is very good given current information it provides to users.

If we move this to export_simple_model_to_onnx.py, it's weird that we emphasize onnxscript version when you don't use it in that code.

This version check honestly doesn't block any code, and it actually provides users another debug (version), if their code doesn't work like tutorial. I really see no harm to keep it.

Reply:
Thanks, I am keeping this for now. We can have more detailed discussion on how to present version check when I am back.